lib/Generator.js

```diff
@@ -20,40 +20,40 @@
       var chmod = Q.denodeify(fs.chmod);
       var mkdir = Q.denodeify(mkdirp);
 
-      //function that handles .gitignore
-      function gitignore(path){
-        return function(){
-          return fs.appendFile('.gitignore', "\n" + path, function(err){
-            if(err) return Promise.reject(err);
-            return Promise.resolve();
-          });
-        };
-      }
 
       // Kick off a promise chain
       mkdir('ansible/inventory', '0755')
-        .then(getReader('ansible.cfg.mustache')).then(render).then(getWriter('ansible/ansible.cfg'))
-        .then(getReader('aws_keys.mustache')).then(render).then(getWriter('ansible/inventory/aws_keys'))
-        .then(gitignore('ansible/inventory/aws_keys'))
-        .then(getReader('destroy-old-nodes.yaml.mustache')).then(render)
-          .then(getWriter('ansible/destroy-old-nodes.yaml'))
-        .then(getReader('ec2.ini.mustache')).then(render).then(getWriter('ansible/inventory/ec2.ini'))
-        .then(getReader('ec2.py')).then(render).then(getWriter('ansible/inventory/ec2.py'))
-        .then(getReader('immutable.yaml.mustache')).then(render).then(getWriter('ansible/immutable.yaml'))
-        .then(getReader('provision.sh.mustache')).then(render).then(getWriter('ansible/provision.sh'))
-          .then(chmod('ansible/provision.sh', '0744'))
-        .then(getReader('tag-old-nodes.yaml.mustache')).then(render).then(getWriter('ansible/tag-old-nodes.yaml'))
+        .then(process('ansible.cfg.mustache', 'ansible/ansible.cfg'))
+        .then(process('aws_keys.mustache', 'ansible/inventory/aws_keys'))
+        .then(process('destroy-old-nodes.yaml.mustache', 'ansible/destroy-old-nodes.yaml'))
+        .then(process('ec2.ini.mustache', 'ansible/inventory/ec2.ini'))
+        .then(process('ec2.py', 'ansible/inventory/ec2.py'))
+        .then(process('immutable.yaml.mustache', 'ansible/immutable.yaml'))
+        .then(process('provision.sh.mustache', 'ansible/provision.sh'))
+        .then(process('tag-old-nodes.yaml.mustache', 'ansible/tag-old-nodes.yaml'))
         .then(mkdir('ansible/roles/' + answers.appName + '/tasks'))
-        .then(getReader('roles/appName/tasks/main.yaml.mustache')).then(render).then(getWriter('ansible/roles/' +
-          answers.appName + '/tasks/main.yaml'))
-        .then(getReader('myApp.pem.mustache')).then(render).then(getWriter('ansible/' + answers.appPem))
-        .then(gitignore('ansible/' + answers.appPem))
```

Comment:
hey is gitignore now gone from the product with this patch? Was that the result of a conflict resolution?